interfaces/builtin/desktop_test.go

```diff
@@ -219,3 +219,42 @@ func (s *DesktopInterfaceSuite) TestStaticInfo(c *C) {
 func (s *DesktopInterfaceSuite) TestInterfaces(c *C) {
 	c.Check(builtin.Interfaces(), testutil.DeepContains, s.iface)
 }
+
+func (s *DesktopInterfaceSuite) TestDisableMountFontCache(c *C) {
+	const mockSnapYaml = `name: desktop-snap
+version: 1.0
+plugs:
+  desktop:
+    mount-font-cache: false
```

Comment:
we also need test cases for this being true, and this being the empty string (though presumably there are pre-existing tests for the empty case so you don't need to add that if those already exist, I didn't check yet)